src/Ushahidi/Modules/V5/Actions/User/Handlers/CreateUserCommandHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Actions\User\Handlers;
+
+use App\Bus\Action;
+use App\Bus\Command\AbstractCommandHandler;
+use App\Bus\Command\Command;
+use Ushahidi\Modules\V5\Actions\User\Commands\CreateUserCommand;
+use Ushahidi\Modules\V5\Repository\User\UserRepository;
+
+class CreateUserCommandHandler extends AbstractCommandHandler
+{
+
+    private $user_repository;
+
+    public function __construct(UserRepository $user_repository)
+    {
+        $this->user_repository = $user_repository;
+    }
+
+    protected function isSupported(Command $command)
+    {
+        assert(
+            get_class($command) === CreateUserCommand::class,
+            'Provided command not supported'
+        );
+    }
+
+
+    /**
+     * run the command handler
+     * @param CreateUserCommand $command
+     * @return int
+     */
+    public function __invoke(Action $command) //: int
+    {
+        $this->isSupported($command);
+        $command->setId(
+            $this->user_repository->create($command->getEntity())
+        );
+    }
+}
```

Comment:
I think we agreed that for command handlers we can actually return the identifier. I'd rather have us doing an explicit return than mutate commands.

Reply:
As we discussed on slack we will update the command bus handler after completing the current merges then we will modify all command handlers to return the id.